subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/TasksReportPerformanceTest.groovy

```diff
@@ -1,72 +0,0 @@
-/*
- * Copyright 2017 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.gradle.performance.regression.corefeature
-
-import org.gradle.performance.AbstractCrossVersionPerformanceTest
-import spock.lang.Unroll
-
-import static org.gradle.performance.regression.corefeature.TasksReportPerformanceTest.TasksReport.BASIC_TASKS_REPORT
-import static org.gradle.performance.regression.corefeature.TasksReportPerformanceTest.TasksReport.DETAILED_TASKS_REPORT
-
-class TasksReportPerformanceTest extends AbstractCrossVersionPerformanceTest {
```

Comment:
We should still be testing this, it is different from "first use" and we used to have lots of problems with the `tasks` task. We don't want to regress there again.